docs/pages/contributors.rst

```diff
@@ -14,28 +14,42 @@ counts, so do not hesitate!
 2. Fork `the repository <https://github.com/tmbo/questionary>`_ on GitHub to
    start making your changes.
 
-3. Write some tests that show the bug is fixed or that the feature works as
+3. Configure development environment.
+
+  a. create python virtual environment, eg
+
+  .. code-block:: console
+
+    virtualenv .env && source .env/bin/activate
+
+  b. Configure development requirements:
+
+  .. code-block:: console
+
+    make develop
+
+4. Write some tests that show the bug is fixed or that the feature works as
    expected.
 
-4. Ensure your code passes the style checks by running
+5. Ensure your code passes the code quality checks by running
 
   .. code-block:: console
 
-    $ black questionary
+    make lint
```

Comment:
```suggestion
    $ make lint
```

Reply:
what is the reason behind it?

Reply:
It is used to indicate that the command should be written in a shell. The `$` is [rendered differently](https://questionary.readthedocs.io/en/stable/pages/contributors.html) in the documentation to make this clear.

Reply:
Yes, that is true. However, this applies only to sphinx. Unfortunately, githib (where lots of developers can look for contribution docs) doesn't render $ sign, which becomes inconvenient to copy the command

Reply:
I prefer keeping the dollar sign because, by convention, it indicates that the command should be run in the shell.

I agree that the way the GitHub copy/paste works is a bit annoying, however, the documentation is intended to be viewed on the documentation website, rather than GitHub.